app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -307,7 +307,13 @@ class HomeActivityTest {
       profileId = profileId1,
       timestampOlderThanOneWeek = false
     )
+    val loginFirstTimeData = profileTestHelper.logIntoUser()
+    val loginSecondTimeData = profileTestHelper.logIntoUser()
```

Comment:
Perhaps move this into a helper to simplify the test changes here? Such as: ``logCurrentUserInTwice()``.

Reply:
done -- added the changes as discussed before